Makefile

```diff
@@ -0,0 +1,5 @@
+lint:
+	swiftlint --fix
+
+format:
+	swiftformat .
```

Comment:
Does this automatically fix invalid formatting? Or does it just throw an error if invalid formatting is detected?